app/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistrationTest.groovy

```diff
@@ -158,13 +158,80 @@ class EtorDomainRegistrationTest extends Specification {
 
     def "Orders endpoint happy path"() {
         given:
+        def mockAuthValidator = Mock(AuthRequestValidator)
+        TestApplicationContext.register(AuthRequestValidator, mockAuthValidator)
         def expected = 200
         def connector = new EtorDomainRegistration()
+        TestApplicationContext.register(EtorDomainRegistration, connector)
+        def req = new DomainRequest()
+        TestApplicationContext.injectRegisteredImplementations()
+
+        when:
+        mockAuthValidator.isValidAuthenticatedRequest(_ as DomainRequest) >> true
+        def res = connector.handleOrders(req)
+        def actual = res.statusCode
+
+        then:
+        actual == expected
+    }
+
+    def "Orders endpoint validator returns false unhappy path" () {
+        given:
+        def mockAuthValidator = Mock(AuthRequestValidator)
+        TestApplicationContext.register(AuthRequestValidator, mockAuthValidator)
+        def expected = 401
+        def connector = new EtorDomainRegistration()
+        TestApplicationContext.register(EtorDomainRegistration, connector)
+        def req = new DomainRequest()
+        TestApplicationContext.injectRegisteredImplementations()
+
+        when:
+        mockAuthValidator.isValidAuthenticatedRequest(_ as DomainRequest) >> false
```

Comment:
I think this should be under the `given:` section.

Reply:
Good catch. I'll do the rest of them as well.